data/members.json

```diff
@@ -8,24 +8,6 @@
             "github": "https://github.com/smlab-niser",
             "website": "https://niser.ac.in/~smishra/",
             "linkedin": ""
-        },
-        {
-            "name": "Aritra Mukhopadhyay",
-            "desc": "Peithonking",
-            "image": "aritra.jpg",
-            "email": "aritra.mukhopadhyay@niser.ac.in",
-            "github": "https://github.com/PeithonKing",
-            "website": "",
-            "linkedin": ""
-        },
-        {
-            "name": "Gayatri",
```

Comment:
We're removing Gayatri?

Reply:
Gayatri informed that they would like to let go off their responsibilities at the club